quaterion/loss/cos_face_loss.py

```diff
@@ -0,0 +1,85 @@
+from typing import Optional
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch import LongTensor, Tensor
+
+from quaterion.loss.group_loss import GroupLoss
+
+
+def l2_norm(inputs: torch.Tensor, dim: int = 0) -> torch.Tensor:
+    """Apply L2 normalization to tensor
+    Args:
+        inputs: Input tensor.
+        dim: Dimension to operate on.
+    Returns:
+        torch.Tensor: L2-normalized tensor
+    """
+    outputs = inputs / torch.norm(inputs, 2, dim, True)
+
+    return outputs
```

Comment:
Hi @crsdvaibhav, there is already a `l2_norm` function implemented for `arcface_loss`. If you are going to use `l2_norm` for `cos_face_loss` as well, then maybe it would be better to take `l2_norm` function to `loss/utils/utils.py` and call it from there in order to avoid code duplication.